tls/s2n_server_key_exchange.c

```diff
@@ -135,6 +136,42 @@ int s2n_dhe_server_key_recv_parse_data(struct s2n_connection *conn, union s2n_ke
     return 0;
 }
 
+int s2n_kem_server_key_recv_read_data(struct s2n_connection *conn, struct s2n_blob *data_to_verify, union s2n_kex_raw_server_data *raw_server_data)
+{
+    struct s2n_kem_raw_server_params *kem_data = &raw_server_data->kem_data;
+    struct s2n_stuffer *in = &conn->handshake.io;
+    const struct s2n_kem *kem = conn->secure.kem_params.negotiated_kem;
+    uint16_t key_length;
+
+    /* Keep a copy to the start of the whole structure for the signature check */
```

Comment:
Please choose one comment style (see line 150).

Reply:
Going with `//` this is a hold of refactoring some other stuff and s2n has mixed one line comment style.